common/utils/src/main/scala/org/apache/spark/internal/Logging.scala

```diff
@@ -127,95 +141,60 @@ trait Logging {
   }
 
   // Log methods that take only a String
-  protected def logInfo(msg: => String): Unit = {
-    if (log.isInfoEnabled) log.info(msg)
-  }
-
-  protected def logInfo(entry: LogEntry): Unit = {
-    if (log.isInfoEnabled) {
-      withLogContext(entry.context) {
-        log.info(entry.message)
-      }
-    }
-  }
-
-  protected def logInfo(entry: LogEntry, throwable: Throwable): Unit = {
-    if (log.isInfoEnabled) {
-      withLogContext(entry.context) {
-        log.info(entry.message, throwable)
-      }
-    }
-  }
-
-  protected def logDebug(msg: => String): Unit = {
-    if (log.isDebugEnabled) log.debug(msg)
-  }
-
-  protected def logTrace(msg: => String): Unit = {
-    if (log.isTraceEnabled) log.trace(msg)
-  }
-
-  protected def logWarning(msg: => String): Unit = {
-    if (log.isWarnEnabled) log.warn(msg)
-  }
-
-  protected def logWarning(entry: LogEntry): Unit = {
-    if (log.isWarnEnabled) {
-      withLogContext(entry.context) {
-        log.warn(entry.message)
-      }
-    }
-  }
-
-  protected def logWarning(entry: LogEntry, throwable: Throwable): Unit = {
-    if (log.isWarnEnabled) {
-      withLogContext(entry.context) {
-        log.warn(entry.message, throwable)
-      }
-    }
-  }
-
-  protected def logError(msg: => String): Unit = {
-    if (log.isErrorEnabled) log.error(msg)
-  }
+  protected def logDebug(msg: => String): Unit = if (log.isDebugEnabled) log.debug(msg)
```

Comment:
Shall we keep the rest of the file unchanged?

Reply:
This is done.